RandBLAS/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Please add a comment indicating why this is needed.

CMake comments start with ``#``.

Reply:
I think I added that because my default compiler had a c++ standard that threw errors with the use of auto in RandBLAS. I removed it and it seems to compile. I can remove that line.